dags/utils/gcp.py

```diff
@@ -397,6 +397,7 @@ def bigquery_etl_query(
     """
     kwargs["task_id"] = kwargs.get("task_id", destination_table)
     kwargs["name"] = kwargs.get("name", kwargs["task_id"].replace("_", "-"))
+    sql_file_path = "sql/{}.sql".format(destination_table)
```

Comment:
We need to build the sql file path _before_ we modify the destination_table to include a date suffix